src/sarif/sarif_file.py

```diff
@@ -28,6 +29,40 @@ def has_sarif_file_extension(filename):
     return any(filename_upper.endswith(x) for x in [".SARIF", ".SARIF.JSON"])
 
 
+def _read_result_location(result) -> Tuple[str, str]:
+    """
+    Extract the file path and line number (latter as a string) from the Result.
```

Comment:
Both as strings or only latter as a string?

Reply:
Both, though that's obvious for the first one.  I've reworded.